CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## 1.2.2 - 2022-12-21
```

Comment:
issue: New version entry will be created automatically by our release process. Could you please move your changelong entry into the Unreleased section?

Reply:
Yes, of course!